client/src/pages/Rides.js

```diff
@@ -49,9 +49,30 @@ const Rides = () => {
         <HeaderMain />
         {rides &&
           rides.map((ride, i) => {
-            return <p key={i}>{ride.start}</p>;
+            return (
+              <Card
+                key={i}
```

Comment:
Because of the general dislike of index-use, maybe something like this would work:
```suggestion
                key={start+dest+rider}
```
We don't have something unique yet... maybe we should generate a UUID for each ride.

Reply:
We actually have something unique in the ID given by our mongoDB. We could consider using that as our key.

Reply:
Changed it, so we now use the id given by mongoDB.

Reply:
like a "baus" 😎